### ISSUE TITLE
Argument for @NotNull parameter 'editor' of com/intellij/openapi/editor/impl/EditorFactoryImpl.releaseEditor must not be null

### ISSUE THREAD

**Issue by petrsimon** [hsz/idea-gitignore#108]
Webstorm  build 141.614. Happened soon after I closed tab with gitignore file.
```
Argument for @NotNull parameter 'editor' of com/intellij/openapi/editor/impl/EditorFactoryImpl.releaseEditor must not be null
java.lang.IllegalArgumentException: Argument for @NotNull parameter 'editor' of com/intellij/openapi/editor/impl/EditorFactoryImpl.releaseEditor must not be null
	at com.intellij.openapi.editor.impl.EditorFactoryImpl.releaseEditor(EditorFactoryImpl.java)
	at mobi.hsz.idea.gitignore.outer.OuterIgnoreWrapper.dispose(OuterIgnoreWrapper.java:96)
	at com.intellij.openapi.util.Disposer$1.execute(Disposer.java:47)
	at com.intellij.openapi.util.Disposer$1.execute(Disposer.java:44)
	at com.intellij.openapi.util.objectTree.ObjectNode$1.execute(ObjectNode.java:132)
	at com.intellij.openapi.util.objectTree.ObjectNode$1.execute(ObjectNode.java:106)
	at com.intellij.openapi.util.objectTree.ObjectTree.executeActionWithRecursiveGuard(ObjectTree.java:146)
	at com.intellij.openapi.util.objectTree.ObjectNode.execute(ObjectNode.java:106)
	at com.intellij.openapi.util.objectTree.ObjectNode$1.execute(ObjectNode.java:122)
	at com.intellij.openapi.util.objectTree.ObjectNode$1.execute(ObjectNode.java:106)
	at com.intellij.openapi.util.objectTree.ObjectTree.executeActionWithRecursiveGuard(ObjectTree.java:146)
	at com.intellij.openapi.util.objectTree.ObjectNode.execute(ObjectNode.java:106)
	at com.intellij.openapi.util.objectTree.ObjectTree.executeAll(ObjectTree.java:132)
	at com.intellij.openapi.util.Disposer.dispose(Disposer.java:108)
	at com.intellij.openapi.util.Disposer.dispose(Disposer.java:104)
	at com.intellij.openapi.fileEditor.impl.text.TextEditorProvider.disposeEditor(TextEditorProvider.java:92)
	at com.intellij.openapi.fileEditor.impl.FileEditorManagerImpl.disposeComposite(FileEditorManagerImpl.java:1649)
	at com.intellij.openapi.fileEditor.impl.EditorWindow$3.run(EditorWindow.java:335)
	at com.intellij.openapi.fileEditor.impl.FileEditorManagerImpl.runChange(FileEditorManagerImpl.java:1679)
	at com.intellij.openapi.fileEditor.impl.EditorWindow.closeFile(EditorWindow.java:316)
	at com.intellij.openapi.fileEditor.impl.FileEditorManagerImpl$6.run(FileEditorManagerImpl.java:600)
	at com.intellij.openapi.command.impl.CoreCommandProcessor.executeCommand(CoreCommandProcessor.java:124)
	at com.intellij.openapi.command.impl.CoreCommandProcessor.executeCommand(CoreCommandProcessor.java:99)
	at com.intellij.openapi.command.impl.CoreCommandProcessor.executeCommand(CoreCommandProcessor.java:85)
	at com.intellij.openapi.fileEditor.impl.FileEditorManagerImpl.closeFile(FileEditorManagerImpl.java:596)
	at com.intellij.openapi.fileEditor.impl.FileEditorManagerImpl.closeFile(FileEditorManagerImpl.java:609)
	at com.intellij.openapi.fileEditor.impl.EditorTabbedContainer$CloseTab.actionPerformed(EditorTabbedContainer.java:446)
	at com.intellij.openapi.keymap.impl.IdeKeyEventDispatcher$3.performAction(IdeKeyEventDispatcher.java:586)
	at com.intellij.openapi.keymap.impl.IdeKeyEventDispatcher.processAction(IdeKeyEventDispatcher.java:637)
	at com.intellij.openapi.keymap.impl.IdeKeyEventDispatcher.d(IdeKeyEventDispatcher.java:476)
	at com.intellij.openapi.keymap.impl.IdeKeyEventDispatcher.dispatchKeyEvent(IdeKeyEventDispatcher.java:212)
	at com.intellij.ide.IdeEventQueue._dispatchEvent(IdeEventQueue.java:538)
	at com.intellij.ide.IdeEventQueue.dispatchEvent(IdeEventQueue.java:382)
	at java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:201)
	at java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:116)
	at java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:105)
	at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:101)
	at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:93)
	at java.awt.EventDispatchThread.run(EventDispatchThread.java:82)
```

**Comment by hsz** [comment]
Thank you for reporting, fixed.

**Comment by hsz** [comment]
Fix has ben released just right now with `v1.1.2`

**Comment by petrsimon** [comment]
Thank you!
